<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MultipleVideoPlayer - PlayList</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="multiplevideoplayer.qrc" >
    <normaloff>:/images/volume.png</normaloff>:/images/volume.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>53</y>
     <width>426</width>
     <height>353</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" colspan="2" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget" >
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <column>
        <property name="text" >
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Artist</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Album</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Year</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2" >
     <widget class="QLabel" name="backendInfosLabel" >
      <property name="text" >
       <string>Backend Infos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionAddFiles" />
    <addaction name="actionOpenDVD" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>406</y>
     <width>426</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>19</y>
     <width>426</width>
     <height>34</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="movable" >
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddPlayFile" />
  </widget>
  <action name="actionAddFiles" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
   </property>
   <property name="text" >
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/volume.png</normaloff>:/images/volume.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/qt.png</normaloff>:/images/qt.png</iconset>
   </property>
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPlay" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text" >
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text" >
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionFullscreen" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/fullscreen.png</normaloff>:/images/fullscreen.png</iconset>
   </property>
   <property name="text" >
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionOpenDVD" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/dvd.png</normaloff>:/images/dvd.png</iconset>
   </property>
   <property name="text" >
    <string>Open DVD</string>
   </property>
  </action>
  <action name="actionAddPlayFile" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/addplay.png</normaloff>:/images/addplay.png</iconset>
   </property>
   <property name="text" >
    <string>Add and Play File</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="multiplevideoplayer.qrc" />
 </resources>
 <connections/>
</ui>
